job_spec/WATER_MAP.yml

```
@@ -120,8 +134,18 @@ WATER_MAP:
         - Ref::hand_fraction
         - --membership-threshold
         - Ref::membership_threshold
-        - --include-flood-depth
-        - Ref::include_flood_depth
+      timeout: TODO
+      vcpu: 1
+      memory: 31600
```

```suggestion
      vcpu: 1
      memory: 126000
```
looks like we upped the memory for water map jobs int he river width spec, so I suspect asf-tools needs a lot of memory for 10m water maps.